floodsub_test.go

```diff
@@ -12,10 +12,10 @@ import (
 
 	host "github.com/libp2p/go-libp2p-host"
 	peer "github.com/libp2p/go-libp2p-peer"
+	protocol "github.com/libp2p/go-libp2p-protocol"
 	swarmt "github.com/libp2p/go-libp2p-swarm/testing"
-	//bhost "github.com/libp2p/go-libp2p/p2p/host/basic"
+
```

Comment:
can we get rid of this empty line?

Reply:
Done in https://github.com/libp2p/go-libp2p-pubsub/pull/181/commits/3c78c237ce2e621b839bf70e3587f0e10c2e46b2 !